client/src/components/Visualiser/VisualiserManager.tsx

```diff
@@ -24,8 +26,8 @@ interface Props {
  *     sliders, etc.).
  */
 const VisualiserInterface: React.FC<Props> = ({ topicTitle }) => {
-  const [animationProgress, setAnimationProgress] = useState<number>(0);
-  const [speed, setSpeed] = useState<number>(0.5);
+  const [timelineComplete, setTimelineComplete] = useState<boolean>(false);
+  const [speed, setSpeed] = useState<number>(0.6);
```

Comment:
Ah... probably best to import this default speed as a constant (TODO for new team?)

Reply:
Agree, saved to that list of small todos we had.